### ISSUE TITLE
Unable to add new elements in list wizard in Contao 5.1

### ISSUE THREAD

**Issue by qzminski** [madeyourday/contao-rocksolid-custom-elements#160]
Hey Martin, the extension does not work properly in Contao 5.1. It is not possible to add new elements to the list wizard due to JS error being thrown here:

https://github.com/madeyourday/contao-rocksolid-custom-elements/blob/master/src/Resources/public/js/be_main.js#L228

<img width="807" alt="CleanShot 2023-02-07 at 11 50 03" src="https://user-images.githubusercontent.com/193483/217224900-afbe0dd0-599d-492b-ab52-7285c05bb851.png">

I am not sure what happens but the drag element is missing the `rsce_list_toolbar_drag` CSS class:

```html
<img class="color-scheme--light" src="/system/themes/flexible/icons/drag.svg" width="16" height="16" alt="Move the item via drag and drop" loading="lazy" title="Move the item via drag and drop">
```

I wanted to create a pull request, but now I wonder whether we should just change the JS to match the selector:

```diff
-toolbar.getFirst('.rsce_list_toolbar_drag').setStyle('display', count < 2 ? 'none' : '');
+toolbar.getFirst('img[src$="drag.svg"]').setStyle('display', count < 2 ? 'none' : '');
```

… or shall we investigate what changed between Contao 4 and 5. Also, it seems that drag and drop no longer works at all (even with the fix above).

Please let me know if I should investigate it further myself, or if you already know what happened and how to fix it 🙂

**Comment by ausi** [comment]
So, only Contao 5.1.0-RC2 is affected and Contao 5.0 works fine, right?

**Comment by qzminski** [comment]
I did not test it on Contao 5.0 to be honest, the only Contao 5 setup I have right now is 5.1.0-RC2.

**Comment by ausi** [comment]
Should be fixed with b14c7142a016185f7da1298335170fc864e74c2c @qzminski please verify.

**Comment by qzminski** [comment]
Ah the markup has changed. It works like a charm now, thanks a lot Martin!
